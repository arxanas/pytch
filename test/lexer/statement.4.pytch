let foo =
  bar(
    baz,
    let qux = 1
    qux, grault
    xyzzy,
    plugh
  )
