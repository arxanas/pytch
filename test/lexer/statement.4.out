'let'
leading whitespace ' '
identifier 'foo'
leading whitespace ' '
'='
trailing newline '\n'
leading whitespace '  '
identifier 'bar'
'('
trailing newline '\n'
leading whitespace '    '
identifier 'baz'
','
trailing newline '\n'
leading whitespace '    '
'let'
leading whitespace ' '
identifier 'qux'
leading whitespace ' '
'='
leading whitespace ' '
integer literal '1'
trailing newline '\n'
the end of a 'let' binding
leading whitespace '    '
identifier 'qux'
','
leading whitespace ' '
identifier 'grault'
trailing newline '\n'
the end of a statement
leading whitespace '    '
identifier 'xyzzy'
','
trailing newline '\n'
leading whitespace '    '
identifier 'plugh'
trailing newline '\n'
leading whitespace '  '
')'
trailing newline '\n'
the end of a 'let' binding
the end of the file
